libs/npm/core/src/hooks/api-subscriber/useApiSubscriberReports.ts

```diff
@@ -85,6 +87,13 @@ export const useApiSubscriberReports = (
         }`,
       );
     },
+    exportReport: (reportId: number) => {
+      return download({
+        url: `/subscriber/reports/${reportId}/export`,
+        method: 'post',
+        fileName: `myreport${reportId}.xlsx`,
```

Comment:
Use the name of the file provided by the endpoint if possible.  It should be `{report.name}.xlsx`.  See above comment.